include/nbl/video/IGPUBuffer.h

```diff
@@ -28,7 +28,21 @@ class NBL_API IGPUBuffer : public asset::IBuffer, public IDeviceMemoryBacked, pu
 		// Vulkan: const VkBuffer*
 		virtual const void* getNativeHandle() const = 0;
 
-		struct SCreationParams : asset::IBuffer::SCreationParams, IDeviceMemoryBacked::SCreationParams {};
+		struct SCreationParams : asset::IBuffer::SCreationParams, IDeviceMemoryBacked::SCreationParams
+		{
+SCreationParams() = default;
+			
+SCreationParams(const asset::IBuffer::SCreationParams& params) 
+	: asset::IBuffer::SCreationParams(params)
+{ }
```

Comment:
dont add the constructors, it disables initializer list initialization like `{}`

Reply:
Ok